src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.stars {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.stars__row {
+  display: flex;
+  gap: 5px;
+}
+
+.stars__star {
+  width: 30px;
+  height: 30px;
+  display: inline-block;
+  background-size: cover;
+  background-image: url('../src/images/star.svg');
+}
```

Comment:
The use of `display: inline-block` is unnecessary here since you are already using `display: flex` in the parent container `.stars__row`. Flex items are automatically block level in the context of a flex container.